tutorials/installation.md

```diff
@@ -46,9 +46,8 @@ Install Gazebo Math:
   brew install gz-math<#>
   ```
 
-Be sure to replace `<#>` with a number value, such as 1 or 2, depending on
-which version you need. From version 7 you should use `gz-math<#>` but for lower versions
-you should use `ign-math<#>`.
+Be sure to replace `<#>` with a number value, such as 6 or 7, depending on
+which version you need.
```

Comment:
Naming is the same in MacOS ?

Reply:
yes, we added `gz-*` aliases for all currently supported packages (dependencies of Citadel, Fortress and gazebo9) in https://github.com/osrf/homebrew-simulation/pull/1984